storm/src/main/storm/mesos/MesosSupervisor.java

```diff
@@ -122,10 +125,12 @@ public String getAssignmentId() {
 
   @Override
   public void killedWorker(int port) {
+    LOG.info("killedWorker: removing port " + port + " from the 'assigned port state'");
+    String taskId = _state.get(Integer.toString(port));
```

Comment:
`Integer.toString(port)` is kind of a non standard way of using `toString` method. I think `String.valueOf(port)` would be better.